src/posit/connect/system.py

```diff
@@ -0,0 +1,292 @@
+        ```python
+        from posit.connect import Client
+
+        client = Client()
+
+        caches = client.system.caches.runtime.find()
+        ```
+
+        """
+        path = self._path + "/caches"
+        return SystemCaches(self._ctx, path)
+
+
+class SystemCaches(ContextManager):
+    """System caches."""
+
+    def __init__(self, ctx: Context, path: str) -> None:
+        super().__init__()
+        self._ctx: Context = ctx
+        # v1/system/caches
+        self._path: str = path
+
+    @property
+    def runtime(self) -> SystemRuntimeCaches:
+        """
+        System runtime caches.
+
+        Returns
+        -------
+        SystemRuntimeCaches
+            Helper class to manage system runtime caches.
+
+        Examples
+        --------
+        ```python
+        from posit.connect import Client
+
+        client = Client()
+
+        caches = client.system.caches.runtime.find()
+        ```
+        """
+        path = self._path + "/runtime"
+        return SystemRuntimeCaches(self._ctx, path)
+
+
+class SystemRuntimeCache(Active):
+    class _Attrs(TypedDict, total=False):
+        language: str  # "Python"
+        """The runtime language of the cache."""
+        version: str  # "3.8.12"
+        """The language version of the cache."""
+        image_name: str  # "Local"
+        """The name of the cache's execution environment."""
+
+    def __init__(
+        self,
+        ctx: Context,
+        path: str,
+        /,
+        **attributes: Unpack[_Attrs],
+    ) -> None:
+        super().__init__(ctx, path, **attributes)
+
+    class _DestroyAttrs(TypedDict, total=False):
+        dry_run: bool
+        """If `True`, the cache will not be destroyed, only the operation will be simulated."""
+
+    @overload
+    def destroy(self, *, dry_run: Literal[True]) -> None: ...
+    @overload
+    def destroy(self, *, dry_run: Literal[False] = False) -> Task: ...
+    def destroy(self, **kwargs) -> Task | None:
+        """
+        Remove a content runtime package cache.
+
+        This action is only available to administrators.
+
+        Parameters
+        ----------
+        dry_run : bool, optional
+            If `True`, the cache will not be destroyed, only the operation will be simulated.
+
+        Returns
+        -------
+        Task | None
+            The task object if the operation was successful. If `dry_run=True`, `None` is returned.
+
+        Examples
+        --------
+        ```python
+        from posit.connect import Client
+
+        client = Client()
+
+        runtime_caches = client.system.caches.runtime.find()
+        first_runtime_cache = runtime_caches[0]
+
+        # Remove the cache
+        task = first_runtime_cache.destroy(dry_run=False)
+
+        # Wait for the task to finish
+        task.wait_for()
+        ```
+        """
+        url = self._ctx.url + self._path
+        data = dict(self)
+        response = self._ctx.session.delete(url, json={**data, **kwargs})
```

Comment:
fwiw, I think this will also work...

```suggestion
        body = {**self, **kwargs}
        response = self._ctx.client.delete(self._path, json=body)
```

Reply:
It doesn't need the full url?

Reply:
The client methods do the path append

Reply:
I missed the switch from `.session` to `.client`. 👍

Reply:
Either one is fine for now. But we could remove _ctx.session in a refactor.

Reply:
I like it a bunch. Making a separate PR that does more of this work.

Reply:
Great. It's a good debt-maintenance task if you have time.

Reply:
Done in #360 

Resolving